### PR TITLE
AST-3571 - GIFs not working with the Header builder

### PR DESCRIPTION
**Pull request by Sudhanshu-1508**
### Description

[Task](https://brainstormforce.atlassian.net/browse/AST-3571)
### Types of changes
Bug Fixes.

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/markup-extras.php [hunk 1]
```diff
@@ -1847,27 +1847,32 @@ function astra_replace_header_attr( $attr, $attachment, $size ) {
 		if ( ! isset( $attachment ) ) {
 			return $attr;
 		}
-
+
```
Comment (imnavanath) [inline inc/markup-extras.php:1850]:
```suggestion

```

File: inc/markup-extras.php [hunk 2]
```diff
@@ -1847,27 +1847,32 @@ function astra_replace_header_attr( $attr, $attachment, $size ) {
 		if ( ! isset( $attachment ) ) {
 			return $attr;
 		}
-
+		
 		$custom_logo_id     = get_theme_mod( 'custom_logo' );
 		$is_logo_attachment = ( $custom_logo_id == $attachment->ID ) ? true : false;
-
+
```
Comment (imnavanath) [inline inc/markup-extras.php:1853]:
```suggestion

```

File: inc/markup-extras.php [hunk 3]
```diff
@@ -1847,27 +1847,32 @@ function astra_replace_header_attr( $attr, $attachment, $size ) {
...
 		$custom_logo_id     = get_theme_mod( 'custom_logo' );
 		$is_logo_attachment = ( $custom_logo_id == $attachment->ID ) ? true : false;
-
+		
 		if ( apply_filters( 'astra_is_logo_attachment', $is_logo_attachment, $attachment ) ) {
 
 			if ( ! is_customize_preview() ) {
 				$attach_data = wp_get_attachment_image_src( $attachment->ID, 'ast-logo-size' );
-
+
```
Comment (imnavanath) [inline inc/markup-extras.php:1858]:
```suggestion

```

File: inc/markup-extras.php [hunk 4]
```diff
@@ -1847,27 +1847,32 @@ function astra_replace_header_attr( $attr, $attachment, $size ) {
...
 		if ( apply_filters( 'astra_is_logo_attachment', $is_logo_attachment, $attachment ) ) {
 
 			if ( ! is_customize_preview() ) {
 				$attach_data = wp_get_attachment_image_src( $attachment->ID, 'ast-logo-size' );
-
+				
 				if ( isset( $attach_data[0] ) ) {
 					$attr['src'] = $attach_data[0];
 				}
+
```
Comment (imnavanath) [inline inc/markup-extras.php:1862]:
```suggestion

```

File: inc/markup-extras.php [hunk 5]
```diff
@@ -1847,27 +1847,32 @@ function astra_replace_header_attr( $attr, $attachment, $size ) {
...
 				if ( isset( $attach_data[0] ) ) {
 					$attr['src'] = $attach_data[0];
 				}
+				
 			}
 
 			$file_type      = wp_check_filetype( $attr['src'] );
 			$file_extension = $file_type['ext'];
-
+
```
Comment (imnavanath) [inline inc/markup-extras.php:1867]:
```suggestion

```

Comment (imnavanath) [reply inc/markup-extras.php:1867]: Remove extra empty space.

**Comment by imnavanath** [comment]
@Sudhanshu-1508 please add changelog for it.

**Comment by Sudhanshu-1508** [comment]
New PR created for the same task due to branch conflicts, closing this one. [https://github.com/brainstormforce/astra/pull/5563](https://github.com/brainstormforce/astra/pull/5563)

**Pull request closed without merging**